src/network/networkprotocol.h

```diff
@@ -207,7 +207,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 		Minimap modes
 	PROTOCOL VERSION 40:
 		TOCLIENT_MEDIA_PUSH changed, TOSERVER_HAVE_MEDIA added
-		AO_CMD_SET_BONE_POSITION extended
+		Added new particlespawner parameters (5.6.0)
```

Comment:
It looks like you merged this incorrectly, the bone position comment is now gone

Reply:
I wanted it to be gone since there is no protocol bump for it (or there will be a protocol bump for 5.7 anyways), should I re-add it?